addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1767,6 +1767,39 @@ options.registry.CoverProperties = options.Class.extend({
     },
 });
 
+/**
+ * allows control over shadows
+ */
+options.registry.shadow = options.Class.extend({
+    /**
+     * @override
+     */
+    selectClass(previewMode, widgetValue, params) {
+        this._super(...arguments);
+        if (!widgetValue) {
+            this.$target.css('box-shadow', '');
+        }
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async _computeWidgetState(methodName, params) {
+        let value = await this._super(...arguments);
+        if (methodName === 'selectStyle' && params.cssProperty === 'box-shadow') {
+            const color = value.slice(value.indexOf('rgb'), value.indexOf(')') + 1);
```

Comment:
Maybe there is more robust to do that this, nothing guarantee the color cannot be returned as an hex code. Maybe check for inset, remove it, then check if the first character is a digit, otherwise it is a color?